delfin/task_manager/scheduler/schedule_manager.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+
+import six
+from oslo_config import cfg
+from oslo_log import log
+from oslo_utils import uuidutils
+
+from delfin import context
+from delfin.task_manager.scheduler import scheduler
+from delfin.task_manager.scheduler.schedulers.telemetry import telemetry_job
+
+LOG = log.getLogger(__name__)
+CONF = cfg.CONF
+
+telemetry_opts = [
+    cfg.IntOpt('periodic_task_schedule_interval', default=180,
```

Comment:
lets use 'job' in the context of scheduler.. what we create and add in scheduler is job

Reply:
Done